src/main/java/com/fasterxml/jackson/core/io/CharTypes.java

```diff
@@ -284,12 +288,12 @@ public static void appendQuoted(StringBuilder sb, String content)
         }
     }
 
-    public static char[] copyHexChars() {
```

Comment:
Would require retaining the old method as deprecated, otherwise this is breaking change. Same for the next method.

Reply:
I kept the old ones as defaulting to uppercase/true